### DISCUSSION TITLE
Data from Instagram

### DISCUSSION THREAD

**Discussion by viajerus** [cjbarrie/academictwitteR#358] [Q&A] [answered]
Hi, I was looking a the JSON response from some tweets and I saw that Instagram is listed as a source. What does this mean? I thought that the package just collects Twitter data. 

<img width="409" alt="Bildschirmfoto 2022-11-21 um 02 06 20" src="https://user-images.githubusercontent.com/63015973/202937812-c05b90fb-a3e0-4d31-9b15-b7557e91c96b.png">

**Comment by cjbarrie** [comment, marked as answer]
It just means this was source of the original content

**Reply by viajerus** [reply]
So the content was re-posted in twitter?
